docs/user-guide/concepts.md

```diff
@@ -218,6 +221,25 @@ probably not a great idea.
 You can create a subnamespace from the command line via `kubectl hns create child
 -n parent`.
 
+<a name="hierarchical-resource-quota">
+
+### Hierarchical Resource Quotas
+
+When you want to give some amount of resources to `team-a`, and want them to be able
+to distrtibute their resources between themselves, you create a HierarchicalResourceQuota
```

Comment:
sp: distribute

Maybe instead of "between themselves," you can say "to be able to flexibly use resources in _any_ of their namespaces"

Reply:
sounds like better english, 
but it also sound like a differente point

i was more trying to tell that they can limit their subnamespaces with hrqs as well (becaues that is exactly why we needed hrqs), 
and not that hrq will work over all of the subnamespaces

for example in this tree:
```
acme-org
├── team-a
│   ├── developer-1
│   └── developer-2
└── team-b
```
`acme-org` will have hrq with 100 cpus, and they can give 60 cpu to `team-a` and 40 cpu to `team-b`,
then `team-a` can give 30 cpu to `developer-1` and other 30 cpus to `developer-2`...
and the whole point is that each section/team can split and distribute their own reasources between themselves without involving the cluster admins.

should i add a chart with example so it would be more clear?

Reply:
IMO most of the value in HRQ is that you _don't_ need to do that if you don't want to. For example, without HRQ, we could give the admin of `team-a` the ability to create `ResourceQuotas` in `team-a`, and they could just create a normal `ResourceQuota` in each of `developer-1` and `developer-2`. What HRQ gives you is the ability to share quota _across_ `developer-1` _and_ `developer-2` - that is, just put a single HRQ object in `team-a` and not worry about the details.

Without HRQ, if `team-a` is allowed to use 60 CPUs, _either_ they need to put an RQ of 60 in both `developer-1` and `developer-2`, which means that the team could actually use up to 120 (oops). Or they need put in RQs of 30 in each, as you suggested, which means that even if `developer-1` is only using 1 CPU and `developer-2` needs to use 50, they can't.

In other words, without HRQ, you have to be either too loose or too strict. With HRQ, you you only need to be strict where it really matters (e.g. at the team level) and everything else will just work.

Does that make sense?

Reply:
> without HRQ, we could give the admin of `team-a` the ability to create ResourceQuotas in `team-a`, and they could just create a normal `ResourceQuota` in each of `developer-1` and `developer-2`.

thats exactly the problem we all struggled with, because if we give to `team-a` the ability to create/manage `ResourceQuotas`, then they have no limit, they can use as much resources as they want.
and the solution is having HRQ in `team-a` namespace that they cannot modify, but they can create HRQs in their subnamespaces (`developer-1` and `developer-2`).
this allows for each team to seperate their resources across their subteams.

i can give more clear example but then it a litle complitaes and has many teams (big tree)

---

> Without HRQ, if team-a is allowed to use 60 CPUs, either they need to put an RQ of 60 in both developer-1 and developer-2, which means that the team could actually use up to 120 (oops). Or they need put in RQs of 30 in each, as you suggested, which means that even if developer-1 is only using 1 CPU and developer-2 needs to use 50, they can't.

thats a good point, but its not what im trying to tell
i think i will try to add both points.
give me 30 min

Reply:
Oh, I see. Hmm. You could still create two RQs in `acme-org` and have one propagate only to `team-a` and the other only to `team-b`, but yes, using HRQs for that is much more elegant.

Reply:
now as im thinking about it im not sure if its possible 😕
is it possible to make admin of `team-a` but be able to manage HRQs in the subnamespaces (`developer-1` and `developer-2`) but not in his own namespace?

admin of `team-a` CANT change HRQs in namespace `team-a`
admin of `team-a` CAN change HRQs in subnamespace `developer-1` and `developer-2`

Reply:
ok, sorry it took me a while to figure out how its possible to make
what do you think of it now?
(idk how to put comments on the text itself, im pretty new at git)